### DISCUSSION TITLE
bond0 interface created manually but suricata still using interface eno2

### DISCUSSION THREAD

**Discussion by Gregg3** [Security-Onion-Solutions/securityonion#9557] [General] [answered]
Installed a Sensor (Forward Node) on physical server from the 2.3.190 iso. During installation, I asked it to monitor traffic on all the interfaces except for the management interface. For some reason, it didn't create bond0. Suricata was only capturing packets from interface eno2. I've manually created bond0 and run so-setup-network. Even tried manually editing the suricata.yaml and the onion2_sensor.sls but nothing that I've tried has been effective. Suricata is still listening on only eno2.

Maybe I made a mistake in adding bond0 with this command:
sudo nmcli con add type bond con-name bond0 ifname bond0 mode 0

What am I doing wrong?
![onion2-bond0](https://user-images.githubusercontent.com/17577820/211838334-198c8965-48b3-4891-9cdb-4f6303645e1b.PNG)

**Comment by robbiemarshall** [comment]
Try running this command and see if that fixes the issue:

nmcli con up bond0-slave-eno2

**Reply by Gregg3** [reply]
Thanks for the suggestion. That command made the interfaces look like what I would expect... but still only getting data from eno2.
![image](https://user-images.githubusercontent.com/17577820/212167037-6d2896a0-a95c-4bb6-8378-1668042de695.png)
I've tried manually editing some configuration files to change eno2 to bond0, but that doesn't seem to have any effect.
Maybe the answer is to configure a separate suricata to handle each network interface?

**Reply by Gregg3** [reply]
In the Onion <Manager/Search>, I can modify /opt/so/saltstack/local/pillar/minions/<ForwardNode>_sensor.sls - and then restart suricata on <ForwardNode>. This allows me to change the interface from eno2 to eno3, but when I change it to bond0, suricata captures nothing.

**Comment by Gregg3** [comment, marked as answer]
We decided to give up on getting suricata to capture from multiple interfaces using bond0. Configured the Cisco switch to combine the interfaces onto one 10G interface. Suricata is now monitoring that traffic using ens3f0.

So, for us, this is a closed issue.
Thanks for the assistance.
